modules/spo-applying-profiles.adoc

```diff
@@ -156,11 +164,19 @@ kind: Pod
 metadata:
   name: nginx-secure
   namespace: nginx-deploy
+  securityContext:
+   runAsNonRoot: true
+   seccompProfile:
+     type: RuntimeDefault
 spec:
   containers:
     - image: nginxinc/nginx-unprivileged:1.21
       name: nginx
+      # TODO: Is this the proper way to integrate this?
```

Comment:
@s-urbaniak @stlaz there already was a securityContext section for the seLinuxOptions. Did I integrate this correctly below?